src/types.d.ts

```diff
@@ -0,0 +1,49 @@
+interface Member {
+  name: string;
+  img: string;
+  title: string;
+}
+
+interface TeamMember extends Member {}
+
+interface BoardMember extends Member {
```

Comment:
add: `type: 'Board'`

Then instead of using the boolean you can j check `type`

Reply:
synced offline!